esphome/components/xiaomi_mccgq02hl/binary_sensor.py

```diff
@@ -0,0 +1,74 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import sensor, binary_sensor, esp32_ble_tracker
+from esphome.const import (
+    CONF_MAC_ADDRESS,
+    CONF_ID,
+    CONF_BINDKEY,
+    CONF_OPEN,
+    CONF_DEVICE_CLASS,
+    CONF_HAS_LIGHT,
+    CONF_BATTERY_LEVEL,
+    UNIT_PERCENT,
+    ICON_BATTERY,
+    DEVICE_CLASS_BATTERY,
+    DEVICE_CLASS_LIGHT,
+)
+
+DEPENDENCIES = ["esp32_ble_tracker"]
+AUTO_LOAD = ["xiaomi_ble", "sensor"]
+
+xiaomi_mccgq02hl_ns = cg.esphome_ns.namespace("xiaomi_mccgq02hl")
+XiaomiMCCGQ02HL = xiaomi_mccgq02hl_ns.class_(
+    "XiaomiMCCGQ02HL",
+    cg.Component,
+    binary_sensor.BinarySensor,
+    esp32_ble_tracker.ESPBTDeviceListener,
+)
+
+CONFIG_SCHEMA = cv.All(
+    binary_sensor.BINARY_SENSOR_SCHEMA.extend(
+        {
+            cv.GenerateID(): cv.declare_id(XiaomiMCCGQ02HL),
+            cv.Required(CONF_MAC_ADDRESS): cv.mac_address,
+            cv.Required(CONF_BINDKEY): cv.bind_key,
+            cv.Optional(CONF_HAS_LIGHT): binary_sensor.BINARY_SENSOR_SCHEMA.extend(
+                {
+                    cv.Optional(DEVICE_CLASS_LIGHT): binary_sensor.device_class,
```

Comment:
The aim here was to use a constant for the default value, not to change the configuration option's name.
```suggestion
                    cv.Optional(CONF_DEVICE_CLASS, default=DEVICE_CLASS_LIGHT): binary_sensor.device_class,
```